modules/storage-ephemeral-vols-overview.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * storage/generic-ephemeral-vols.adoc
+
+:_content-type: CONCEPTS
+[id="generic-ephemeral-vols-overview_{context}"]
+= Overview
+
+Generic ephemeral volumes are a type of ephemeral volume that can be provided by all storage drivers that also support persistent volumes and dynamic provisioning. Generic ephemeral volumes are similar to `emptyDir` volumes in that they provide a per-pod directory for scratch data, which is usually empty after provisioning. Generic ephemeral volumes are specified inline in the pod spec.
```

Comment:
Consider changing the first sentence to Active voice (can be provided is Passive voice)